src/components/FocusCard.jsx

```diff
@@ -1,7 +1,75 @@
 import React from "react";
+import { FaGears } from "react-icons/fa6";
+import { GoPeople } from "react-icons/go";
+import { RiComputerLine } from "react-icons/ri";
 
 const FocusCard = () => {
-  return <div>FocusCard</div>;
+  return (
+    <div
+      className="flex flex-wrap text-white"
+      style={{ justifyContent: "center" }}
+    >
+      <div
+        className="flex flex-col justify-between bg-asme-blue-200"
+        style={{ margin: "3.5%", width: "289px", height: "289px" }}
```

Comment:
use tailwind instead of style, so you can just write in the className that m-[3.5%] w-[289px] h-[289px]